# generated by patch-package 6.4.7 on 2021-07-12 13:17:16
#
# command:
#   npx patch-package idb
#
# declared package:
#   @dimensiondev/kit: 0.0.0-20211215044858-27d964f
#
diff --git a/node_modules/@dimensiondev/kit/package.json b/node_modules/@dimensiondev/kit/package.json
index 153848e..98a26e6 100644
--- a/node_modules/@dimensiondev/kit/package.json
+++ b/node_modules/@dimensiondev/kit/package.json
@@ -4,7 +4,7 @@
   "description": "A toolkit library.",
   "license": "MIT",
   "author": "Septs",
-  "main": "./cjs/index.cjs",
+  "main": "./esm/index.js",
   "module": "./esm/index.js",
   "type": "module",
   "exports": {
